src/experimental/ProbabilisticGraphicalModels/ProbabilisticGraphicalModels.jl

```diff
@@ -3,16 +3,33 @@ module ProbabilisticGraphicalModels
 using BangBang
 using Graphs
 using Distributions
+using JuliaBUGS
+using JuliaBUGS: BUGSGraph, VarName
+using JuliaBUGS.MetaGraphsNext
+
+import Graphs: edges
```

Comment:
Generally, it's a good idea to be explicit when using these functions. What I mean here is, instead of importing `edges`, use `Graphs.edges` in the code. The reason is that in this way, it is always clear which function one is using. Of course, in case a function is used multiple times, it might make sense to do `using` on a package level. 

Also use `using` instead of `import` unless you need to overload it.